rpc/lighthouse.go

```diff
@@ -514,26 +514,46 @@ func (lc *LighthouseClient) GetBlocksBySlot(slot uint64) ([]*types.Block, error)
 	if payload := parsedBlock.Message.Body.ExecutionPayload; payload != nil && !bytes.Equal(payload.ParentHash, make([]byte, 32)) {
 		txs := make([]*types.Transaction, 0, len(payload.Transactions))
 		for _, rawTx := range payload.Transactions {
-			tx := &types.Transaction{Raw: rawTx}
+			// TODO: Decide how to deal with fields that can't be obtained.
+			//       DB expects non nil values. Currently just defaulting everything to 0.
```

Comment:
Empty byte slices should work fine. It's `nil` that doesn't work with the postgres lib (and the fields really shouldn't be nil, with the exception of `To` on contract deploy transactions)

Reply:
See my comment in here: https://github.com/protolambda/eth2-beaconchain-explorer/pull/3#issuecomment-989304116